crds/zz_defs.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by go-bindata. (@generated) DO NOT EDIT.
 
- //Package crds generated by go-bindata.// sources:
```

Comment:
Do we want to be formatting generated files? Can we adjust the fmt target to filter out certain files/directories/etc.?

Reply:
IMO if the generator isn't formatting Go files it has a bug, so I'm fine with formatting everything (with the exception of vendor dirs).